src/universal/utils/fromNow.js

```diff
@@ -26,3 +26,16 @@ export default function fromNow(time) {
   // this is both for eslint, and for chuckles. It should never happen:
   return 'infinitely long ago';
 }
+
+export function refreshPeriod(time) {
+  const msElapsed = (Date.now() - time) || 0;
+  const threshKeys = Object.keys(thresholds);
+  for (let i = 1; i < threshKeys.length; i++) {
```

Comment:
Fascinating. I learned that even `return` does not short-circuit a forEach loop. Great point, and accepted!